WalletWasabi.Gui/Program.cs

```diff
@@ -26,7 +28,18 @@ private static void Main(string[] args)
 			bool runGui = false;
 			try
 			{
-				Global = new Global();
+				string dataDir = EnvironmentHelpers.GetDataDir(Path.Combine("WalletWasabi", "Client"));
+				Directory.CreateDirectory(dataDir);
+				string torLogsFile = Path.Combine(dataDir, "TorLogs.txt");
+
+				var uiConfig = new UiConfig(Path.Combine(dataDir, "UiConfig.json"));
+				uiConfig.LoadOrCreateDefaultFile();
+				var config = new Config(Path.Combine(dataDir, "Config.json"));
+				config.LoadOrCreateDefaultFile();
+				config.CorrectMixUntilAnonymitySet();
+				var walletManager = new WalletManager(config.Network, new WalletDirectories(dataDir));
+
+				Global = new Global(dataDir, torLogsFile, config, uiConfig, walletManager);
```

Comment:
Please extract this lines to a builder method.

Reply:
Done